packages/grant-explorer/src/features/round/ViewProjectDetails.tsx

```diff
@@ -105,26 +111,93 @@ function ProjectTitle(props: { projectMetadata: ProjectMetadata }) {
 function AboutProject(props: { projectToRender: Project }) {
 
   const { projectToRender } = props;
+  const projectRecipient = projectToRender.recipient.slice(0, 6) + "..." + projectToRender.recipient.slice(-4);
+  const projectWebsite = projectToRender.projectMetadata.website;
+  const projectTwitter = projectToRender.projectMetadata.projectTwitter;
+  const userGithub = projectToRender.projectMetadata.userGithub;
+  const projectGithub = projectToRender.projectMetadata.projectGithub;
 
   return (
-    <div className="border-b-2 pt-2 pb-6">
-      <Detail text={projectToRender.projectMetadata.website} />
-      <Detail text={`@${projectToRender.projectMetadata.projectTwitter}`} />
+    <div className="grid grid-cols-2 border-b-2 pt-2 pb-6">
+      {projectRecipient && 
+        (<span className="flex items-center mt-4 gap-1">
+          <LightningBoltIcon className="h-4 w-4 mr-1 opacity-40" />
+          <DetailSummary text={`${projectRecipient}`} testID="project-recipient" />
+        </span>)
+      }
+      {projectWebsite && 
+        (<span className="flex items-center mt-4 gap-1">
+          <GlobeAltIcon className="h-4 w-4 mr-1 opacity-40" />
+          <a 
+            href={projectWebsite} 
+            target="_blank" 
+            rel="noreferrer" 
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary text={`${projectWebsite}`} testID="project-website" />
+          </a>
+        </span>)
+      }
+      {projectTwitter && 
+        (<span className="flex items-center mt-4 gap-1">
+          <TwitterIcon className="h-4 w-4 mr-1 opacity-40" />
+          <a 
+            href={`https://twitter.com/${projectTwitter}`}
+            target="_blank" 
+            rel="noreferrer" 
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary text={`${projectTwitter}`} testID="project-twitter" />
```

Comment:
from the design, the twitter handle should have '@' prefix

i.e `@nfrgosselin` not `nfrgosselin`

Reply:
i've fixed this ✅